app/services/most_popular_content.rb

```diff
@@ -27,16 +25,15 @@ def fetch
 
   def search_response
     params = {
-      start: 0,
-      count: number_of_links,
-      fields: RummagerFields::TAXON_SEARCH_FIELDS,
-      filter_part_of_taxonomy_tree: content_ids,
-      order: '-popularity',
-      reject_link: current_path,
+        start: 0,
+        count: number_of_links + 1,
```

Comment:
Does `+1` assume that you'd only ever receive one link back that matches the current path? I don't know if this works with multipage formats like guides (see PR fix we did previously here: https://github.com/alphagov/government-frontend/pull/1026)

Reply:
Ah, right, yes, thanks for letting me know. can't think of a way around this without either making another rummager call with the reject added back in to make up the numbers, or just requesting >1 extra links (in which case, how can I figure out how many is a good number?) - do you know any promising directions to go in?

Reply:
For multipage formats, so long as they're only one content item (which I think is true), rummager will only know about them once.  So I think we're fine with the `+ 1` approach.